docs/build_docs.sh

```diff
@@ -54,7 +54,7 @@ JEKYLL_CMD="build"
 while getopts ":p" opt; do
 	case $opt in
 		p)
-		JEKYLL_CMD="serve --config _config.yml,_local_preview_conf.yml --watch"
+		JEKYLL_CMD="serve --baseurl= --watch"
```

Comment:
Thanks! Just tried this out and it works as expected. I guess it was there, because older Jekyll versions didn't have support for this. @dyanarose, do know exactly what changed (if anything) between Jekyll versions?

+1 to apply this change after #2033 has been merged.